contracts/exchange-libs/src/reference_functions.ts

```diff
@@ -2,10 +2,10 @@ import { ReferenceFunctions } from '@0x/contracts-utils';
 import { FillResults, Order } from '@0x/types';
 import { BigNumber } from '@0x/utils';
 
-import LibMathRevertErrors = require('./lib_math_revert_errors');
-
 const { safeAdd, safeSub, safeMul, safeDiv } = ReferenceFunctions;
 
+import { LibMathRevertErrors } from '../src';
```

Comment:
Please do not import from `../src` anywhere. This makes future refactors difficult and is an anti-pattern.

Reply:
Why is it an anti-pattern? It's also testing that it is actually exported in the `index.ts` for consumers.

Reply:
I'm not sure why it's considered an anti-pattern, but it does seem cleaner and more readable to just import from `@0x/utils` to me.